polkadot/node/core/pvf/src/host.rs

```diff
@@ -887,17 +893,17 @@ fn pulse_every(interval: std::time::Duration) -> impl futures::Stream<Item = ()>
 /// We do this check by spawning a new process and trying to sandbox it. To get as close as possible
 /// to running the check in a worker, we try it... in a worker. The expected return status is 0 on
 /// success and -1 on failure.
-fn check_can_unshare_user_namespace_and_change_root(
+async fn check_can_unshare_user_namespace_and_change_root(
 	#[cfg_attr(not(target_os = "linux"), allow(unused_variables))]
 	prepare_worker_program_path: &Path,
 ) -> bool {
 	cfg_if::cfg_if! {
 		if #[cfg(target_os = "linux")] {
-			let output = std::process::Command::new(prepare_worker_program_path)
+			match tokio::process::Command::new(prepare_worker_program_path)
```

Comment:
why did we switch to using tokio here?

Reply:
Previously we were running these checks sequentially. So, waiting for each process to spawn and return before running the next check. We can instead do this concurrently for a small performance gain. It's only relevant on node startup, but every 0.1 s regression adds up over time.